pkg/util/testing/error_matchers.go

```diff
@@ -67,6 +67,7 @@ func BeError(errorType ErrorType) types.GomegaMatcher {
 func (matcher *errorMatcher) Match(actual any) (success bool, err error) {
 	err, ok := actual.(error)
 	if !ok {
+		//nolint:staticcheck // This is a user-facing error, so we keep it as is.
```

Comment:
This is not user-facing actually, it is just for tests

Reply:
If we look at the official matchers (https://github.com/onsi/gomega/blob/59bd7f9a8908fce4705a9003b7c8e1ce51c08a2c/matchers/succeed_matcher.go#L24-L26), the error message starts with an uppercase letter. So I’d prefer to keep it as is for consistency.

In result we have:

```
  [FAILED] Error matcher expects an error.  Got:
      <nil>: nil
  In [It] at: /Users/mykhailo_bobrovskyi/Projects/epam/kueue/testbin/test/test_test.go:12 @ 04/25/25 09:55:45.452
```

Reply:
Maybe, but saying this is user-facing error is misleading. I'm ok to say "we keep the error capitalized for consistency with built-in matchers".

Reply:
Agree. Thank you! Changed explanation.